### PR TITLE
[Hub Generated] Review request for Microsoft.KeyVault to add version stable/2019-09-01

### PR DESCRIPTION
**Pull request by heaths**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/Azure/azure-rest-api-specs/dev-keyvault-Microsoft.KeyVault-2019-09-01...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/keyvault.json [hunk 1]
```diff
@@ -1122,7 +1122,13 @@
         },
         "enableSoftDelete": {
           "type": "boolean",
-          "description": "Property to specify whether the 'soft delete' functionality is enabled for this key vault. If omitted, assume true as default value. Once set to true, cannot be reverted to false."
+          "default": true,
+          "description": "Property to specify whether the 'soft delete' functionality is enabled for this key vault. If it's not set to any value(true or false) when creating new key vault, it will be set to true by default. Once it's been set to true value, it can NOT be reverted to false."
+        },
+        "softDeleteRetentionInDays": {
```
Comment (majastrz) [inline specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/keyvault.json]: >softDeleteRetentionInDays [](start = 9, length = 25)

What's the default value on `softDeleteRetentionInDays` and can we put it in the swagger?

Comment (gdlxm) [reply specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/keyvault.json]: 90

Comment (majastrz) [reply specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2019-09-01/keyvault.json]: Thanks for updating

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

**Review by schaabs — APPROVED** [review]

**Comment by heaths** [comment]
The validation passed, I checked the two checkboxes in the issue description which retriggered the pipelines, then they failed (i.e. no code changes). @akning-ms can your team look into this?

**Comment by heaths** [comment]
/azp run automation - sdk

Timeout in JavaScript SDK generation - the other ones that previous failed without code change passed.

**Pull request merged**